internal/commands/config/get_internal.go

```diff
@@ -32,7 +33,20 @@ func RunInternalConfigGet(viperKey string) (err error) {
 			return fmt.Errorf("failed to get configuration: %v", err)
 		}
 
-		if opt.Sensitive {
+		var unmaskValues bool
+		unmaskOptionVal, err := profiles.GetOptionValue(options.ConfigUnmaskSecretValueOption)
+		if err != nil {
+			unmaskValues = false
+		} else {
+			unmaskValuesBool, err := strconv.ParseBool(unmaskOptionVal)
+			if err != nil {
+				unmaskValues = false
+			} else {
+				unmaskValues = unmaskValuesBool
+			}
+		}
+
+		if opt.Sensitive && !unmaskValues {
```

Comment:
Consider simplifying here to use the string directly instead of casting to bool.
`strings.EqualFold()`